Source/Events.Store.MongoDB/Streams/Filters/AbstractFilterDefinition.cs

```diff
@@ -36,6 +38,16 @@ protected AbstractFilterDefinition(Guid filterId, Guid sourceStream)
         /// </summary>
         public Guid SourceStream { get; set; }
```

Comment:
I believe this property should not be present on the filter definition, as it always has to be the Event Log.

Reply:
Removed all of the properties from `AbstractFilterDefinitions`.